types/sanitize-html/index.d.ts

```diff
@@ -12,64 +12,56 @@
-    allowedSchemes: string[];
-    allowedSchemesByTag: { [index: string]: string[] };
-    allowedTags: string[];
-    selfClosing: string[];
-  }
-
-
-  interface IFrame {
-    tag: string;
-    attribs: { [index: string]: string };
-    text: string;
-    tagPosition: number;
-  }
-
-
-  interface IOptions {
-    allowedAttributes?: { [index: string]: AllowedAttribute[] } | boolean;
-    allowedStyles?:  { [index: string]: { [index: string]: RegExp[] } };
-    allowedClasses?: { [index: string]: string[] } | boolean;
-    allowedIframeHostnames?: string[];
-    allowIframeRelativeUrls?: boolean;
-    allowedSchemes?: string[] | boolean;
-    allowedSchemesByTag?: { [index: string]: string[] } | boolean;
-    allowedSchemesAppliedToAttributes?: string[];
-    allowProtocolRelative?: boolean;
-    allowedTags?: string[] | boolean;
-    textFilter?: (text: string) => string; 
-    exclusiveFilter?: (frame: IFrame) => boolean;
-    nonTextTags?: string[];
-    selfClosing?: string[];
-    transformTags?: { [tagName: string]: string | Transformer };
-    parser?: Options;
-  }
-
-
-  var defaults: IDefaults;
-
-
-  function simpleTransform(tagName: string, attribs: Attributes, merge?: boolean): Transformer;
+    type Attributes = { [attr: string]: string };
+
+    type Tag = { tagName: string; attribs: Attributes; text?: string };
+
+    type Transformer = (tagName: string, attribs: Attributes) => Tag;
+
+    type AllowedAttribute = string | { name: string; multiple?: boolean; values: string[] };
+
+    interface IDefaults {
+        allowedAttributes: { [index: string]: AllowedAttribute[] };
+        allowedSchemes: string[];
+        allowedSchemesByTag: { [index: string]: string[] };
+        allowedTags: string[];
+        selfClosing: string[];
+    }
+
+    interface IFrame {
+        tag: string;
+        attribs: { [index: string]: string };
+        text: string;
+        tagPosition: number;
+    }
+
+    interface IOptions {
+        allowedAttributes?: { [index: string]: AllowedAttribute[] } | boolean;
+        allowedStyles?: { [index: string]: { [index: string]: RegExp[] } };
+        allowedClasses?: { [index: string]: string[] } | boolean;
+        allowedIframeHostnames?: string[];
+        allowIframeRelativeUrls?: boolean;
+        allowedSchemes?: string[] | boolean;
+        allowedSchemesByTag?: { [index: string]: string[] } | boolean;
+        allowedSchemesAppliedToAttributes?: string[];
+        allowProtocolRelative?: boolean;
+        allowedTags?: string[] | boolean;
+        textFilter?: (text: string) => string;
+        exclusiveFilter?: (frame: IFrame) => boolean;
+        nonTextTags?: string[];
+        selfClosing?: string[];
+        transformTags?: { [tagName: string]: string | Transformer };
+        parser?: Options;
+    }
+
+    var defaults: IDefaults;
+
+    function simpleTransform(tagName: string, attribs: Attributes, merge?: boolean): Transformer;
```

Comment:
I'm all for the indenting tidy up, but it makes it very hard to see what actually changed, if anything, in here. It's easier for reviewers if you submit formatting changes in a separate PR, then make the actual changes after. I know it's hard to keep to that though, I use Intellij's Reformat Code all the time, and sometimes on an older file it does a lot of work!

For the purpose of this PR, please could you add comments highlighting any actual changes in there.

Reply:
Undid all the formatting changes; good call.